src/js/components/DataFilters/DataFilters.js

```diff
@@ -23,43 +30,61 @@ const layerProps = {
   position: 'right',
 };
 
-export const DataFilters = ({ drop, children, heading, layer, ...rest }) => {
+const defaultTouched = {};
+export const DataFilters = ({
+  drop,
+  children,
+  showClearFilters = true,
```

Comment:
@halocline @jcfilben @MikeKingdom thoughts on this prop name?

Reply:
any reason not to do something simple like `clear`?

Reply:
Some of the initial thinking was the prop is really around whether or not to "show" the control -- sort of similar to Calendar's `showAdjacentDays` prop or SelectMultiple `showSelectedInline`. Even though the prop name is a bit more verbose, it feels like it's a little more clear on what it means. My worry with something like `clear` is that it's unclear if it's functioning as an action (if set to true, does it clear out the filters?) etc.

Reply:
Would like to hear @jcfilben and @MikeKingdom 's reaction to this rationale.

Reply:
Agree with leaning towards being more verbose/increased clarity rather than having a shorter prop name. I'm good with `showClearFilters`

Reply:
After discussion, we will rename to `clearFilters` and remove `show` language. In general, grommet tries to avoid use of verbs/leading words in variable names to keep them simple. It also allows us better flexibility in the future if we want to extend this prop to accept more than just a boolean. Updated to reflected this.